### PR TITLE
[SRNPU/NNFW] Add NNFW Type for SRNPU

### PR DESCRIPTION
**Pull request by dongju-chae**
This patch adds NNFW Type for SRNPU.

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

### CODE REVIEW

**Comment by again4you** [comment]
@dongju-chae 
This patch needs an ACR process. If it is not _urgent_, what about merging this just after Tizen 6.5 M1 release?

File: c/include/nnstreamer.h [hunk 1]
```diff
@@ -141,6 +141,7 @@ typedef enum {
   ML_NNFW_TYPE_PYTORCH = 11,          /**< PyTorch (.pt). (Since 6.5) */
   ML_NNFW_TYPE_NNTR_INF = 12,         /**< Inference supported from NNTrainer, SR On-device Training Framework (Since 6.5) */
   ML_NNFW_TYPE_SNAP = 0x2001,         /**< SNAP (Samsung Neural Acceleration Platform), only for Android. (Since 6.0) */
+  ML_NNFW_TYPE_SRNPU = 0x3000,        /**< SR (Samsung Research) made NPU (.tvn). (Since 6.5) */
```
Comment (jaeyun-jung) [inline c/include/nnstreamer.h:144]: plz set ```ML_NNFW_TYPE_SRNPU == 13```

**Comment by dongju-chae** [comment]
> This patch needs an ACR process. If it is not urgent, what about merging this just after Tizen 6.5 M1 release?

No problem :)

File: c/include/nnstreamer.h (hunk 1 above)
Comment (dongju-chae) [reply c/include/nnstreamer.h]: Updated, thanks.

**Review by myungjoo — APPROVED** [review]
LGTM

File: c/include/nnstreamer.h (hunk 1 above)
Comment (jaeyun-jung) [reply c/include/nnstreamer.h]: Oh..I didnt check this PR, can you rebase and udpate with new enum value ML_NNFW_TYPE_SRNPU = 14

**Review by myungjoo — CHANGES_REQUESTED** [review]
NAME CONFLICT WITH ANOTHER PR (MERGED). REBASE AND UPDATE THE NUMBER

**Comment by dongju-chae** [comment]
Got it, Updated.

**Review by myungjoo — APPROVED** [review]

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**